src/user-points/user-points.jobs.controller.ts

```diff
@@ -68,10 +74,69 @@ export class UserPointsJobsController {
     }
 
     const options = await this.eventsService.getUpsertPointsOptions(user);
-    await this.userPointsService.upsert(options);
+    const userPoints = await this.userPointsService.upsert(options);
+
+    if (
+      userPoints.pool1_points === null ||
+      userPoints.pool2_points === null ||
+      userPoints.pool3_points === null
+    ) {
+      const refreshPreviousPoolOptions: RefreshPreviousPoolOptions = {
```

Comment:
Shouldn't we be getting points for pull requests from all phases?

Reply:
We have points for pull requests from previous phases in the current db